[docs] [migrate] Performance section for YB Voyager (#13003)
yugabyte/yugabyte-db: docs/content/preview/migrate/yb-voyager/_index.md

> 

Reviewer: Replace the first 2 paragraphs with:

YugabyteDB Voyager is a powerful open-source data migration engine that accelerates cloud native adoption by removing barriers to moving applications to the public or private cloud. It helps you migrate databases to YugabyteDB quickly and securely.

YugabyteDB Voyager manages the entire lifecycle of a database migration, including cluster preparation for data import, schema-migration, and data-migration, using the [yb-voyager](https://github.com/yugabyte/yb-voyager) CLI.

## Features

> - It’s free and completely open source.

Reviewer: ```suggestion
- Free and completely open source.
```

> - The engine supports widely used databases for migration and does not require changes to the [source databases](#source-databases) in most cases.

Reviewer: ```suggestion
- Supports widely used databases for migration and doesn't require changes to the [source databases](#source-databases) in most cases.
```

> - All YugabyteDB products (v2.12 and above) are supported as a [target database](#target-database).

Reviewer: ```suggestion
- Supports all YugabyteDB products (v2.12 and above) as the [target database](#target-database).
```

> - A unified [CLI](../yb-voyager/yb-voyager-cli/) experience for all different source databases.

Reviewer: ```suggestion
- Provides a unified [CLI](../yb-voyager/yb-voyager-cli/) experience for all different source databases.
```

> - It's auto-tunable based on workloads, by analyzing the target cluster capacity. By default, it runs parallel jobs.

Reviewer: ```suggestion
- Auto-tuneable based on workloads, by analyzing the target cluster capacity; runs parallel jobs by default.
```

      </div>
    </a>
  </div>
   <div class="col-12 col-md-6 col-lg-12 col-xl-6">
    <a class="section-link icon-offset" href="performance/">
      <div class="head">
       <img class="icon" src="/images/section_icons/explore/high_performance.png" aria-hidden="true">
        <div class="title">Performance</div>
      </div>
      <div class="body">

>         Learn about the factors affecting export and import performance.

Reviewer: ```suggestion
        Learn about factors that can affect migration performance.
```
